pkg/client/ocm/types.go

```diff
@@ -23,3 +23,10 @@ const (
 const (
 	RHOSAKResourceName string = "rhosak"
 )
+
+type AMSRelatedResourceBillingModel string
```

Comment:
Can we make `var supportedAMSRelatedResourceBillingModels = map[string]struct{}{` consistent with this?

Also a `String` method can be added to this type to avoid doing manual type conversion